airbyte-local-cli-nodejs/src/utils.ts

```diff
@@ -296,3 +313,28 @@ export function processSrcInputFile(tmpDir: string, cfg: FarosConfig): Promise<v
     });
   });
 }
+
+export function generateDstStreamPrefix(cfg: FarosConfig): void {
+  const srcImage = cfg.src?.image;
+  const dstImage = cfg.dst?.image;
+  if (dstImage?.startsWith('farosai/airbyte-faros-destination')) {
+    // if source image is a faros feed image
+    if (
+      !cfg.connectionName &&
+      srcImage?.startsWith('farosai/airbyte-faros-feeds-source') &&
+      (cfg.src?.config as any)?.feed_cfg?.feed_name
+    ) {
+      cfg.connectionName = `${(cfg.src?.config as any)?.feed_cfg?.feed_name}-feed`;
+      logger.debug(`Using connection name: ${cfg.connectionName}`);
+    }
+    // if image is an airbyte image
+    if (srcImage?.startsWith('farosai/airbyte')) {
+      const [imageName] = srcImage.split(':');
+      const imageParts = imageName?.split('-').slice(1, -1);
+      cfg.connectionName = `my${imageParts?.join('') ?? ''}src`;
+      cfg.dstStreamPrefix = `${cfg.connectionName}_${imageParts?.join('_') ?? ''}__`;
```

Comment:
```suggestion
      cfg.dstStreamPrefix = `${cfg.connectionName}__${imageParts?.join('_') ?? ''}__`;
```

Reply:
Needs to be two underscores here so that the faros destination can distinguish the connectionName (origin) from the imageParts (sourceType).